src/components/atoms/Alert/Alert.js

```diff
@@ -41,30 +47,37 @@ export default class Alert extends HTMLElement {
     this.shadowRoot.appendChild(bootStyles);
     this.shadowRoot.appendChild(variableStyles);
     this.shadowRoot.appendChild(alertStyles);
-    // alert attributes
-
-    const icon = this.getAttribute('data-icon');
-
-    const iconOrder = this.getAttribute('data-icon-order');
 
-    const iconSize = this.getAttribute('data-icon-size');
-
-    const backgroundColor = this.getAttribute('data-background-color');
-
-    const extraClasses = this.getAttribute('data-extra-classes');
+    // alert attributes
+    // TODO: See CityOfDetroit/detroitmi#1099
+    // eslint-disable-next-line prefer-const
+    let icon = this.getAttribute('data-icon');
+    // TODO: See CityOfDetroit/detroitmi#1099
+    // eslint-disable-next-line prefer-const
+    let iconOrder = this.getAttribute('data-icon-order');
+    // TODO: See CityOfDetroit/detroitmi#1099
+    // eslint-disable-next-line prefer-const
+    let iconSize = this.getAttribute('data-icon-size');
+    // TODO: See CityOfDetroit/detroitmi#1099
+    // eslint-disable-next-line prefer-const
+    let backgroundColor = this.getAttribute('data-background-color');
+    // TODO: See CityOfDetroit/detroitmi#1099
+    // eslint-disable-next-line prefer-const
+    let extraClasses = this.getAttribute('data-extra-classes');
     this.alert.role = 'alert';
     let iconClass = '';
-
-    // TODO: Fix old ESLint errors - see issue #1099
+    // TODO: See CityOfDetroit/detroitmi#1099
     // eslint-disable-next-line eqeqeq
     if (icon != undefined && icon != null) {
       this.alert.querySelector('#alert-content').className = 'col';
-
-      const activeIcon = document.createElement('cod-icon');
+      // TODO: See CityOfDetroit/detroitmi#1099
+      // eslint-disable-next-line prefer-const
+      let activeIcon = document.createElement('cod-icon');
       activeIcon.setAttribute('data-icon', icon);
       activeIcon.setAttribute('data-size', iconSize);
-
-      const iconContainer = document.createElement('div');
+      // TODO: See CityOfDetroit/detroitmi#1099
+      // eslint-disable-next-line prefer-const
+      let iconContainer = document.createElement('div');
```

Comment:
I think the revert that you mentioned was a revert to the wrong commit. Take a look at this file on dev to see what these sections should look like: https://github.com/CityOfDetroit/COD-Design-System/blob/dev/src/components/atoms/Alert/Alert.js